backend/tasks/celery_tasks.py

```diff
@@ -109,13 +110,32 @@ def calc_item_stats(self, session=None):
 
 
 @celery.task(base=DBTask, bind=True, priority=9)
-def create_training_pack(self, id_, n=10, date_start=None, date_end=None, session=None):
+def create_training_pack(self, requester_id, pack_player_id, name=None, n=10, date_start=None, date_end=None,
```

Comment:
can we add a comment to differentiate create_training_pack from create_custom_training_pack?